sources/install/package_wordlists.sh

```diff
@@ -106,6 +105,70 @@ function install_genusernames() {
     add-to-list "genusernames,https://gitlab.com/-/snippets/2480505/raw/main/bash,GenUsername is a Python tool for generating a list of usernames based on a name or email address."
 }
 
+
+
+function install_onelistforall() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing onelistforall"
+    wget https://raw.githubusercontent.com/six2dez/OneListForAll/main/onelistforallmicro.txt -P /opt/lists/
+    wget https://raw.githubusercontent.com/six2dez/OneListForAll/main/onelistforallshort.txt -P /opt/lists/
+    add-test-command "[[ -f '/opt/lists/onelistforallshort.txt' ]]"
+    add-to-list "onelistforall,https://github.com/six2dez/OneListForAll,Rockyou for web fuzzing"
+}
+
+function install_rules(){
```

Comment:
We probably don't want to unzip the files here for two reasons
1. it gets a bit heavy (72M)
2. hashcat already support on-the-fly loading of compressed wordlists (https://github.com/hashcat/hashcat/blob/ba2746228a40ebeed53a187dfa54fc6154234481/docs/changes.txt#L14), I guess it works with rules as well

Reply:
Okay, i remove all gunzip, but with `add-test-command` i keep it for checking zip files? @ShutdownRepo